src/backend/distributed/planner/multi_logical_planner.c

```diff
@@ -368,89 +370,117 @@ ErrorIfQueryNotSupported(Query *queryTree)
 	bool hasComplexJoinOrder = false;
 	bool hasComplexRangeTableType = false;
 	bool preconditionsSatisfied = true;
+	char *errorHint = NULL;
+	char *joinHint = "Consider joining tables on partition column and have equal "
```

Comment:
actually conts is not okay here. Later I need to set char * variable, compiler throws warnings on that. I should either cast it back to (char *) or leave it as it is now.